jekyll/_cci2/deployment-integrations.md

```diff
@@ -142,137 +142,44 @@ To deploy to Azure, use a similar job to the above example that uses an appropri
 
 ## Heroku
 
-The built-in Heroku integration through the CircleCI UI is not implemented for CircleCI 2.0.
-However, it is possible to deploy to Heroku manually.
-
-First, create a bash script to set up Heroku and place the file in the `.circleci` directory:
-
- ```bash
- #!/bin/bash
- wget https://cli-assets.heroku.com/branches/stable/heroku-linux-amd64.tar.gz
- sudo mkdir -p /usr/local/lib /usr/local/bin
- sudo tar -xvzf heroku-linux-amd64.tar.gz -C /usr/local/lib
- sudo ln -s /usr/local/lib/heroku/bin/heroku /usr/local/bin/heroku
-
- cat > ~/.netrc << EOF
- machine api.heroku.com
-   login $HEROKU_LOGIN
-   password $HEROKU_API_KEY
- EOF
-
- cat >> ~/.ssh/config << EOF
- VerifyHostKeyDNS yes
- StrictHostKeyChecking no
- EOF
- ```
-
-**Note:** `sudo` is only necessary if the script runs in a Docker container with a non-root user.
-If the user _is_ root,
-remove `sudo` as it may not be installed.
-
-Next, install and authorize Heroku for the CircleCI account that owns the project.
-[Add environment variables]({{ site.baseurl }}/2.0/env-vars/) for the Heroku API key and login email to the CircleCI application as shown in the following image:
-
-![Add Environment Variables]({{ site.baseurl }}/assets/img/docs/walkthrough5.png)
+[Heroku](https://www.heroku.com/) is a popular platform
+for hosting applications in the cloud.
+To configure CircleCI
+to deploy your application to Heroku,
+follow the steps below.
 
-To connect to the Heroku Git server from CircleCI,
-create a new SSH key without a passphrase.
-The private key is added through the SSH Permissions page
-with a hostname of `git.heroku.com` as shown in the following image:
+1. Create a Heroku account
+and [follow the instructions](https://devcenter.heroku.com/start)
```

Comment:
and follow the instructions-> and follow the [Getting Started on Heroku] documentation